lib/QubitCsvTransform.class.php

```diff
@@ -147,10 +147,14 @@ public function addRowToMySQL($sortorder)
     public static function numberedFilePathVariation($filename, $number)
     {
         $parts = pathinfo($filename);
-        $base = $parts['filename'];
-        $path = $parts['dirname'];
 
-        return $path.'/'.$base.'_'.sprintf('%04d', $number).'.'.$parts['extension'];
+        return sprintf(
+            '%s/%s_%s.%s',
+            $parts['dirname'],
+            $parts['filename'],
+            sprintf('%04d', $number),
```

Comment:
This doesn't need a nested sprintf(), you can do:

```
return sprintf(
            '%s/%s_%04d.%s',
            $parts['dirname'],
            $parts['filename'],
            $number,
            $parts['extension']
);
```

Reply:
Ahhh! Slick! Good call.